ovve_ui/utils/comms_link.py

```diff
@@ -316,6 +317,13 @@ def run(self) -> None:
             self.logger.debug('Serial Init Successful')
         else:
             self.logger.error('Serial Initialization failed')
-            # When the alarm infrastructure is done this would trigger an alarm
+            # Signal the UI to display and sound an alarm
+            self.lost_comms_signal.emit()
```

Comment:
This actually might be a good use of the Hardware failure alarm on the UI-CU side; though maybe in the future